### ISSUE TITLE
Operator emit negative values

### ISSUE THREAD

**Issue by samphunter** [pawn-lang/compiler#413]
### Issue description:

Some emit instructions that should take negative integers don't. For example ADDR.pri -20, LOAD.S.pri -20, STOR.S.pri -20, ...

### Minimal complete verifiable example (MCVE):

```
main() {
	emit STACK -20; // Allocate 5 cells
	emit ADDR.alt -20; // Get address of this array of 5 cells
        emit ZERO.pri;
	emit FILL 20; // Fill it with zeros
	emit STOR.S.pri -20; // Access first element
	emit STACK 20; // Free
}
```

### Workspace Information:

* Compiler version: 3.10.9
* Command line arguments provided (or sampctl version): -;+ -(+
* Operating System: Windows 8.1

**Comment by YashasSamaga** [comment]
`#emit` supports negative values but the `emit` operator doesn't.

@Daniel-Cortez

**Comment by Daniel-Cortez** [comment]
Already did the relevant changes locally, I'll probably upload another batch of improvements for `emit`, tomorrow, but again, it would rely on #393 being merged (I wonder what still keeps it from being merged), as I've added a new token (`teREFERENCE`).

**Comment by Daniel-Cortez** [comment]
Fixed in #421.
